geometry_msgs/Pose placePos
---
bool isSetFinsh